src/Models/TableItem.php

```diff
@@ -69,4 +71,61 @@ public function getCMSFields()
 
         return $fields;
     }
+
+    /**
+     * Get a row of cells
+     *
+     * Retrieve cells for this row in the table. Only the number of cells
+     * configured in the containing TableBlock will be returned.
+     *
+     * @return ArrayList
+     */
+    public function getCells()
+    {
+        $cells = [];
+        foreach (range(1, $this->TableBlock->NumberOfColumns) as $cell) {
+            $cells[] = DBField::create_field('HTMLText', $this->{"Cell" . $cell});
+        }
+        $cells = ArrayList::create($cells);
+        return $cells;
+    }
+
+    /**
+     * Check if the FirstColumnIsHeader is checked and is the first row
+     *
+     * @return Boolean
```

Comment:
`Boolean` would refer to a class called Boolean. The type you should use here is `bool`

https://docs.phpdoc.org/guide/references/phpdoc/types.html